test/cases/parsing/harmony-export-import-specifier/index.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+
+import {x, y} from "./a";
+
+it("namespace export as from commonjs should override named export", function() {
```

Comment:
@sokra @domenic
Actually I'm not sure what the spec is here. Should namespace export override the named export?

Reply:
No, the spec says `*` export should have lower precedence than direct export. Order of exports doesn't matter here.

So the correct result would be x = 1